### PR TITLE
Remove RobotBuilder comments from code

### LINKED ISSUE #13: Remove RobotBuilder comments in code

### PR DESCRIPTION
**Pull request by fishyfishfish55**

### CODE REVIEW

File: src/main/java/org/frc571/bradley/commands/AutonomousCommand.java [hunk 1]
```diff
@@ -1,50 +1,21 @@
...
 package org.frc571.bradley.commands;
 
 import edu.wpi.first.wpilibj2.command.CommandBase;
 
-// BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=IMPORTS
 import org.frc571.bradley.subsystems.Drive;
 
-    // END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=IMPORTS
-
 /**
```
Comment (sonicbhoc) [inline src/main/java/org/frc571/bradley/commands/AutonomousCommand.java]: - [x] Remove empty comment.

File: src/main/java/org/frc571/bradley/commands/DriveCommand.java [hunk 2]
```diff
@@ -1,56 +1,29 @@
...
-// BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=IMPORTS
-import org.frc571.bradley.subsystems.Drive;
-
 import edu.wpi.first.wpilibj2.command.CommandBase;
 
-// END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=IMPORTS
-
 /**
  *
  */
```
Comment (sonicbhoc) [inline src/main/java/org/frc571/bradley/commands/DriveCommand.java]: - [x] Remove empty comment.

File: src/main/java/org/frc571/bradley/commands/EjectCommand.java [hunk 3]
```diff
@@ -1,49 +1,21 @@
...
 import edu.wpi.first.wpilibj2.command.CommandBase;
 
-// BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=IMPORTS
 import org.frc571.bradley.subsystems.Intake;
 
-// END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=IMPORTS
-
 /**
  *
  */
```
Comment (sonicbhoc) [inline src/main/java/org/frc571/bradley/commands/EjectCommand.java]: - [x] Remove empty comment

File: src/main/java/org/frc571/bradley/commands/IndexCommand.java [hunk 4]
```diff
@@ -1,49 +1,21 @@
...
 import edu.wpi.first.wpilibj2.command.CommandBase;
 
-// BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=IMPORTS
 import org.frc571.bradley.subsystems.Index;
 
-// END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=IMPORTS
-
 /**
  *
  */
```
Comment (sonicbhoc) [inline src/main/java/org/frc571/bradley/commands/IndexCommand.java]: - [x] Remove empty comment

File: src/main/java/org/frc571/bradley/commands/IntakeCommand.java [hunk 5]
```diff
@@ -1,44 +1,25 @@
...
 import org.frc571.bradley.subsystems.Index;
 import org.frc571.bradley.subsystems.Intake;
 
 import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
 
-// END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=IMPORTS
-
 /**
  *
  */
```
Comment (sonicbhoc) [inline src/main/java/org/frc571/bradley/commands/IntakeCommand.java]: - [x] Remove empty comment

File: src/main/java/org/frc571/bradley/commands/ShootCommand.java [hunk 6]
```diff
@@ -1,49 +1,21 @@
...
 import edu.wpi.first.wpilibj2.command.CommandBase;
 
-// BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=IMPORTS
 import org.frc571.bradley.subsystems.Shooter;
 
-// END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=IMPORTS
-
 /**
  *
  */
```
Comment (sonicbhoc) [inline src/main/java/org/frc571/bradley/commands/ShootCommand.java]: - [x] Remove empty comment

File: src/main/java/org/frc571/bradley/commands/StopIntakeCommand.java [hunk 7]
```diff
@@ -1,49 +1,21 @@
...
 import edu.wpi.first.wpilibj2.command.CommandBase;
 
-// BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=IMPORTS
 import org.frc571.bradley.subsystems.Intake;
 
-// END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=IMPORTS
-
 /**
  *
  */
```
Comment (sonicbhoc) [inline src/main/java/org/frc571/bradley/commands/StopIntakeCommand.java]: - [x] Remove empty comment

File: src/main/java/org/frc571/bradley/commands/SuckCommand.java [hunk 8]
```diff
@@ -1,49 +1,21 @@
...
+
 import org.frc571.bradley.subsystems.Intake;
 
 import edu.wpi.first.wpilibj2.command.CommandBase;
 
-// END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=IMPORTS
-
 /**
  *
  */
```
Comment (sonicbhoc) [inline src/main/java/org/frc571/bradley/commands/SuckCommand.java]: - [x] Remove empty comment

File: src/main/java/org/frc571/bradley/subsystems/Drive.java [hunk 9]
```diff
@@ -1,61 +1,37 @@
...
-// BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=IMPORTS
 import edu.wpi.first.wpilibj.drive.DifferentialDrive;
 import edu.wpi.first.wpilibj.motorcontrol.MotorControllerGroup;
 import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
 
-// END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=IMPORTS
-
 /**
  *
  */
```
Comment (sonicbhoc) [inline src/main/java/org/frc571/bradley/subsystems/Drive.java]: - [x] Remove empty comments

File: src/main/java/org/frc571/bradley/subsystems/Drive.java [hunk 10]
```diff
@@ -1,61 +1,37 @@
...
     private WPI_TalonFX rFollower;
     private MotorControllerGroup rightMotors;
     private DifferentialDrive differentialDrive;
     private WPI_Pigeon2 pigeon;
 
-    // END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=DECLARATIONS
-
     /**
     *
     */
```
Comment (sonicbhoc) [inline src/main/java/org/frc571/bradley/subsystems/Drive.java]: - [x] Remove empty comment.

File: src/main/java/org/frc571/bradley/subsystems/Index.java [hunk 11]
```diff
@@ -1,56 +1,31 @@
...
-
 package org.frc571.bradley.subsystems;
 
 import edu.wpi.first.wpilibj.motorcontrol.PWMVictorSPX;
 
-// END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=IMPORTS
-
 /**
  *
  */
```
Comment (sonicbhoc) [inline src/main/java/org/frc571/bradley/subsystems/Index.java]: - [x] Remove empty comment.

File: src/main/java/org/frc571/bradley/subsystems/Index.java [hunk 12]
```diff
@@ -1,56 +1,31 @@
...
-    // END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=CONSTANTS
-
-    // BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=DECLARATIONS
     private PWMVictorSPX indexMotor;
 
-    // END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=DECLARATIONS
-
     /**
     *
     */
```
Comment (sonicbhoc) [inline src/main/java/org/frc571/bradley/subsystems/Index.java]: - [x] Remove empty comment.

File: src/main/java/org/frc571/bradley/subsystems/Intake.java [hunk 13]
```diff
@@ -1,57 +1,36 @@
...
-
 package org.frc571.bradley.subsystems;
 
 import edu.wpi.first.wpilibj.motorcontrol.PWMVictorSPX;
 
-// END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=IMPORTS
-
 /**
  *
  */
```
Comment (sonicbhoc) [inline src/main/java/org/frc571/bradley/subsystems/Intake.java]: - [x] Remove empty comment.

File: src/main/java/org/frc571/bradley/subsystems/Shooter.java [hunk 14]
```diff
@@ -1,49 +1,27 @@
...
-
 package org.frc571.bradley.subsystems;
 
 import edu.wpi.first.wpilibj.motorcontrol.PWMVictorSPX;
 
-// END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=IMPORTS
-
 /**
  *
  */
```
Comment (sonicbhoc) [inline src/main/java/org/frc571/bradley/subsystems/Shooter.java]: - [x] Remove empty comment.

File: src/main/java/org/frc571/bradley/subsystems/Shooter.java [hunk 15]
```diff
@@ -1,49 +1,27 @@
...
-    // END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=CONSTANTS
-
-    // BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=DECLARATIONS
     private PWMVictorSPX shootMotor;
 
-    // END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=DECLARATIONS
-
     /**
     *
     */
```
Comment (sonicbhoc) [inline src/main/java/org/frc571/bradley/subsystems/Shooter.java]: - [x] Remove empty comment.

File: src/main/java/org/frc571/bradley/subsystems/Shooter.java (hunk 14 above)
Comment (sonicbhoc) [inline src/main/java/org/frc571/bradley/subsystems/Shooter.java]: - [x] Remove empty comment.

File: src/main/java/org/frc571/bradley/subsystems/Shooter.java (hunk 15 above)
Comment (sonicbhoc) [inline src/main/java/org/frc571/bradley/subsystems/Shooter.java]: - [x] Remove empty comment.

**Review by sonicbhoc — CHANGES_REQUESTED** [review]
Lots of empty documentation comments. Please either remove them. Documentation may be done when the code is finalized.

**Comment by fishyfishfish55** [comment]
Removed empty documentation comments

**Review by sonicbhoc — CHANGES_REQUESTED** [review]
A few comments were missed. Please make a second pass.

**Review by sonicbhoc — APPROVED** [review]

**Pull request merged**